### DISCUSSION TITLE
更大程度的样式自定义

### DISCUSSION THREAD

**Discussion by Konfido** [walinejs/waline#274] [Q&A] [answered]
感觉"Login"的位置大小设置地很别扭，预览的icon和位置也可以再调整。如果默认样式暂时没法改，能否支持修改radius，元素border有无等细节。
![image](https://user-images.githubusercontent.com/25447610/118791210-137f9e00-b8c9-11eb-85dd-467146dfacca.png)
个人更倾向下面这种紧凑棱角感：
<img width="859" alt="Xnip2021-05-18_14-36-56" src="https://user-images.githubusercontent.com/25447610/118791869-b33d2c00-b8c9-11eb-9939-a0202f183f12.png">

**Comment by Mister-Hope** [comment]
样式后续版本还会考虑优化，图中的图片按钮，以及诸如表情的多选项卡支持也在考虑中。

目前你可以自行书写CSS覆盖默认样式。就像 https://github.com/lizheming/waline/issues/268#issuecomment-841994567 说的，默认样式几乎没有使用 !important 选择器，你可以很方便的覆盖它。

**Comment by Mister-Hope** [comment, marked as answer]
目前已经提供了无样式版本，你可以自行创建样式。
